includes/class-stats.php

```diff
@@ -309,42 +401,69 @@ private function get_registered_stats() {
 				'job_listing_view'                 => [
 					'log_callback' => [ $this, 'log_stat' ], // Example of overriding how we log this.
 					'trigger'      => 'page-load',
+					'type'         => 'pageLoad',
+					'page'         => 'listing',
 				],
 				'job_listing_view_unique'          => [
-					'unique'          => true,
-					'unique_callback' => [ $this, 'unique_by_post_id' ],
-					'trigger'         => 'page-load',
+					'unique'  => true,
+					'type'    => 'pageLoad',
+					'trigger' => 'page-load',
+					'page'    => 'listing',
 				],
 				'job_listing_apply_button_clicked' => [
-					'trigger'         => 'apply-button-clicked',
-					'element'         => 'input.application_button',
-					'event'           => 'click',
-					'unique'          => true,
-					'unique_callback' => [ $this, 'unique_by_post_id' ],
+					'trigger' => 'apply-button-clicked',
+					'type'    => 'domEvent',
+					'element' => 'input.application_button',
+					'event'   => 'click',
+					'unique'  => true,
+					'page'    => 'listing',
+				],
+				'jobs_view'                        => [
+					'trigger' => 'page-load',
+					'type'    => 'pageLoad',
+					'page'    => 'jobs',
+				],
+				'jobs_view_unique'                 => [
+					'trigger' => 'page-load',
+					'type'    => 'pageLoad',
+					'page'    => 'jobs',
+					'unique'  => true,
+				],
+				'job_listing_impressions'          => [
```

Comment:
Let's make this name singular, `job_listing_impressions` → `job_listing_impression`, like the other events. Plus should we include `search` in the name, since we are planning to have an alert impression too? `job_listing_search_impression`, `job_listing_alert_impression`?

Maybe we should make all the event names a bit shorter, just to save database space? `job_listing` → `job`? Like `job_search_impression`, job_apply_click`.

Reply:
I will let y'all decide about names, changing to singular for now.